src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -4,29 +4,29 @@
 
 public class StorageImpl<K, V> implements Storage<K, V> {
     private static final int ARRAY_SIZE = 10;
-    private Object[] key;
-    private Object[] value;
-    private int sizeTracker = 0;
+    private Object[] keys;
+    private Object[] values;
+    private int sizeTracker;
 
     public StorageImpl() {
-        this.key = new Object[ARRAY_SIZE];
-        this.value = new Object[ARRAY_SIZE];
+        this.keys = new Object[ARRAY_SIZE];
+        this.values = new Object[ARRAY_SIZE];
     }
 
     @Override
     public void put(K key, V value) {
         if (getIndex(key) != -1) {
-            this.value[getIndex(key)] = value;
+            this.values[getIndex(key)] = value;
```

Comment:
```suggestion
            values[getIndex(key)] = value;
```
when you changed arrays names there is no need to use `this` anymore